samples/v1.3/ElementsPreview/Input.Text.Style.json

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.3",
+	"body": [
+		{
+			"type": "Input.Text",
+			"id": "myComment",
+			"label": "Comment (style: text)",
+			"style": "text"
+		},
+		{
+			"type": "Input.Text",
+			"id": "myEmail",
+			"placeholder": "john.doe@adaptivecards.io",
```

Comment:
>placeholder [](start = 4, length = 11)

Do we still want to use placeholders in the new cards? I had thought they were being discouraged in general.